### PR TITLE
Fix comparing selenium tests execution results with CI

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
It fixes parameter of execution of selenium tests **--compare-with-ci** of webdriver.sh.

### What issues does this PR fix or reference?
#6443

### CODE REVIEW

File: selenium/che-selenium-test/selenium-tests.sh [hunk 1]
```diff
@@ -14,19 +14,29 @@ export CALLER=$(basename $0)
...
 
-mvn clean dependency:unpack-dependencies \
-    -DincludeArtifactIds=che-selenium-core \
-    -DincludeGroupIds=org.eclipse.che.selenium \
-    -Dmdep.unpack.includes=webdriver.sh \
-    -DoutputDirectory=${CUR_DIR}/target/bin
-chmod +x target/bin/webdriver.sh
-
 TESTS_SCOPE="--suite=CheSuite.xml"
+CLEAN_TARGET_COMMAND="clean"
```
Comment (tolusha) [inline selenium/che-selenium-test/selenium-tests.sh]: `CLEAN_GOAL`

File: selenium/che-selenium-core/bin/webdriver.sh [hunk 2]
```diff
@@ -529,18 +530,12 @@ fetchActualResults() {
         ACTUAL_RESULTS_URL=${BASE_ACTUAL_RESULTS_URL}${job}"/"
     fi
 
-    # from 'Failed Tests:' to 'Skipped Tests:'
+    # get list of failed tests from CI server
     local actualResults=($(curl -s ${ACTUAL_RESULTS_URL} | \
                            tr '>' '\n' | tr '<' '\n' | tr '"' '\n'  | \
-                           grep -A9999999 "Failed Tests:" | grep -B9999999 "Skipped Tests:" | \
```
Comment (tolusha) [inline selenium/che-selenium-core/bin/webdriver.sh:535]: Do they include skipped tests in Test result? Otherwise we lost something.

Comment (tolusha) [reply selenium/che-selenium-core/bin/webdriver.sh:535]: I am afraid, but url that is used to request failed tests `https://ci.codenvycorp.com/view/qa/job/che-integration-tests/52/` doesn't include all failed tests. The correct  url is `https://ci.codenvycorp.com/view/qa/job/che-integration-tests/52/#showFailuresLink`
Does this PR cover this usecase?

Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:535]: Did you try to curl both urls? :-)

Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:535]: Result is the same:
`ndp@ndp:~$ curl -s https://ci.codenvycorp.com/view/qa/job/che-integration-tests/52/ | \
                            tr '>' '\n' | tr '<' '\n' | tr '"' '\n'  | \
                            grep -A9999999 "Test Result" | \
                            grep [a-z][a-z0-9_]*[.][a-z] | grep -v http | grep -v junit`
```
/static/4c76111a/plugin/embeddable-build-status/images/24x24/shield.png
/static/4c76111a/images/24x24/previous.png
/static/4c76111a/images/24x24/next.png
/static/4c76111a/images/48x48/blue.png
/static/4c76111a/images/48x48/notepad.png
/static/4c76111a/images/48x48/orange-square.png
/static/4c76111a/plugin/git/icons/git-48x48.png
    var elms = document.getElementsByClassName(
 elms.length; i++) {
    elms[i].style.display = 
    elm = document.getElementById(
    elm.style.display = 
/static/4c76111a/images/48x48/clipboard.png
org.eclipse.che.selenium.miscellaneous.CheckRestoringWorkspaceAfterStoppingWsAgentProcess.checkRestoreIdeItems
org.eclipse.che.selenium.miscellaneous.CheckRestoringWorkspaceAfterStoppingWsAgentProcess.checkRestoreWsAgentByApi
org.eclipse.che.selenium.git.GitCompareTest.checkGitCompareTest
org.eclipse.che.selenium.editor.autocomplete.AutocompleteProposalJavaDocTest.shouldReflectChangesInJavaDoc
org.eclipse.che.selenium.subversion.LockFileTest.lockFileTest
org.eclipse.che.selenium.dashboard.RenameWorkspaceTest.renameNameWorkspaceTest
org.eclipse.che.selenium.assistant.KeyBindingsTest.enterKeyCombinationTest
org.eclipse.che.selenium.opendeclaration.Eclipse0121Test.test0121
org.eclipse.che.selenium.factory.DirectUrlFactoryWithSpecificBranch.factoryWithDirectUrlWithSpecificBranch
org.eclipse.che.selenium.mavenplugin.CheckMavenPluginTest.excludeIncludeModules
org.eclipse.che.selenium.refactor.move.MoveJavaClassToSubpackageTest.checkProjectTreeAfterMoveJavaFile
org.eclipse.che.selenium.refactor.packages.RenamePackageSpringTest.checkRenamePackageWithMainMethod
org.eclipse.che.selenium.refactor.packages.RenamePackageSpringTest.checkRenamePackageSpringApp
org.eclipse.che.selenium.filewatcher.EditFilesWithTabsTest.checkEditingFilesFromIde
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTest3
org.eclipse.che.selenium.opendeclaration.Eclipse0115Test.test0115
org.eclipse.che.selenium.miscellaneous.FileStructureNodesTest.checkFileStructureNodes
org.eclipse.che.selenium.miscellaneous.FindUsagesBaseOperationTest.checkFindUsagesBaseOperation
org.eclipse.che.selenium.languageserver.CheckMainFeatureForCharpLanguageTest.checkLaunchingCodeserver
org.eclipse.che.selenium.mavenplugin.CheckMavenPluginTest.shouldAccessClassCreatedInAnotherModuleAfterIncludingModule
org.eclipse.che.selenium.filewatcher.CheckFileWatcherExcludeFeatureTest.checkIsNotExcludeOperationEventAboutIgnoreFile
org.eclipse.che.selenium.debugger.PhpProjectDebuggingTest.shouldDebugWebPhpScriptFromNonDefaultPortAndNotFromFirstLine
org.eclipse.che.selenium.debugger.PhpProjectDebuggingTest.shouldDebugCliPhpScriptFromFirstLine
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkRecentlyCreatedFiles
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkFindWholeWordOnly
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkFindTextBasic
org.eclipse.che.selenium.factory.DirectUrlFactoryWithRootFolder.factoryWithDirectUrlWithRootFolder
org.eclipse.che.selenium.testrunner.JavaTestPluginTestNgTest.shouldExecuteTestMethodsSuccessfully
org.eclipse.che.selenium.plainjava.PlainJavaProjectConfigureClasspathTest.checkConfigureClasspathPlainJavaProject
org.eclipse.che.selenium.subversion.UpdateToRevisionTest.updateToRevisionTest
org.eclipse.che.selenium.workspaces.ProjectStateAfterRenameWorkspaceTest.checkProjectAfterRenameWs
org.eclipse.che.selenium.filewatcher.RefactoringFeatureTest.checkRefactorFilesFromIde
org.eclipse.che.selenium.subversion.DiffViewTest.checkDiffView
org.eclipse.che.selenium.testrunner.JavaTestPluginJunit4Test.shouldExecuteJUnit4MethodWithDifferentStatuses
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTestPackageRenameWithResource12
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTest4
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTestHierarchical9_1
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTestDisableImport11
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkFindIntoSelectedPath
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkFindByFileMask
org.eclipse.che.selenium.editor.CheckRestoringOfSplitEditor.checkRestoringStateSplittedEditor
org.eclipse.che.selenium.debugger.CppProjectDebuggingTest.shouldDebugCppProject
org.eclipse.che.selenium.plainjava.ConfigureSomeSourceFoldersTest.checkConfigureClasspathPlainJavaProject
org.eclipse.che.selenium.debugger.DebugExternalClassTest.shouldDebugJreClass
```
`ndp@ndp:~$ curl -s https://ci.codenvycorp.com/view/qa/job/che-integration-tests/52/#showFailuresLink | \
                            tr '>' '\n' | tr '<' '\n' | tr '"' '\n'  | \
                            grep -A9999999 "Test Result" | \
                            grep [a-z][a-z0-9_]*[.][a-z] | grep -v http | grep -v junit`
```
/static/4c76111a/plugin/embeddable-build-status/images/24x24/shield.png
/static/4c76111a/images/24x24/previous.png
/static/4c76111a/images/24x24/next.png
/static/4c76111a/images/48x48/blue.png
/static/4c76111a/images/48x48/notepad.png
/static/4c76111a/images/48x48/orange-square.png
/static/4c76111a/plugin/git/icons/git-48x48.png
    var elms = document.getElementsByClassName(
 elms.length; i++) {
    elms[i].style.display = 
    elm = document.getElementById(
    elm.style.display = 
/static/4c76111a/images/48x48/clipboard.png
org.eclipse.che.selenium.miscellaneous.CheckRestoringWorkspaceAfterStoppingWsAgentProcess.checkRestoreIdeItems
org.eclipse.che.selenium.miscellaneous.CheckRestoringWorkspaceAfterStoppingWsAgentProcess.checkRestoreWsAgentByApi
org.eclipse.che.selenium.git.GitCompareTest.checkGitCompareTest
org.eclipse.che.selenium.editor.autocomplete.AutocompleteProposalJavaDocTest.shouldReflectChangesInJavaDoc
org.eclipse.che.selenium.subversion.LockFileTest.lockFileTest
org.eclipse.che.selenium.dashboard.RenameWorkspaceTest.renameNameWorkspaceTest
org.eclipse.che.selenium.assistant.KeyBindingsTest.enterKeyCombinationTest
org.eclipse.che.selenium.opendeclaration.Eclipse0121Test.test0121
org.eclipse.che.selenium.factory.DirectUrlFactoryWithSpecificBranch.factoryWithDirectUrlWithSpecificBranch
org.eclipse.che.selenium.mavenplugin.CheckMavenPluginTest.excludeIncludeModules
org.eclipse.che.selenium.refactor.move.MoveJavaClassToSubpackageTest.checkProjectTreeAfterMoveJavaFile
org.eclipse.che.selenium.refactor.packages.RenamePackageSpringTest.checkRenamePackageWithMainMethod
org.eclipse.che.selenium.refactor.packages.RenamePackageSpringTest.checkRenamePackageSpringApp
org.eclipse.che.selenium.filewatcher.EditFilesWithTabsTest.checkEditingFilesFromIde
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTest3
org.eclipse.che.selenium.opendeclaration.Eclipse0115Test.test0115
org.eclipse.che.selenium.miscellaneous.FileStructureNodesTest.checkFileStructureNodes
org.eclipse.che.selenium.miscellaneous.FindUsagesBaseOperationTest.checkFindUsagesBaseOperation
org.eclipse.che.selenium.languageserver.CheckMainFeatureForCharpLanguageTest.checkLaunchingCodeserver
org.eclipse.che.selenium.mavenplugin.CheckMavenPluginTest.shouldAccessClassCreatedInAnotherModuleAfterIncludingModule
org.eclipse.che.selenium.filewatcher.CheckFileWatcherExcludeFeatureTest.checkIsNotExcludeOperationEventAboutIgnoreFile
org.eclipse.che.selenium.debugger.PhpProjectDebuggingTest.shouldDebugWebPhpScriptFromNonDefaultPortAndNotFromFirstLine
org.eclipse.che.selenium.debugger.PhpProjectDebuggingTest.shouldDebugCliPhpScriptFromFirstLine
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkRecentlyCreatedFiles
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkFindWholeWordOnly
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkFindTextBasic
org.eclipse.che.selenium.factory.DirectUrlFactoryWithRootFolder.factoryWithDirectUrlWithRootFolder
org.eclipse.che.selenium.testrunner.JavaTestPluginTestNgTest.shouldExecuteTestMethodsSuccessfully
org.eclipse.che.selenium.plainjava.PlainJavaProjectConfigureClasspathTest.checkConfigureClasspathPlainJavaProject
org.eclipse.che.selenium.subversion.UpdateToRevisionTest.updateToRevisionTest
org.eclipse.che.selenium.workspaces.ProjectStateAfterRenameWorkspaceTest.checkProjectAfterRenameWs
org.eclipse.che.selenium.filewatcher.RefactoringFeatureTest.checkRefactorFilesFromIde
org.eclipse.che.selenium.subversion.DiffViewTest.checkDiffView
org.eclipse.che.selenium.testrunner.JavaTestPluginJunit4Test.shouldExecuteJUnit4MethodWithDifferentStatuses
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTestPackageRenameWithResource12
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTest4
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTestHierarchical9_1
org.eclipse.che.selenium.refactor.packages.RenamePackageTest.checkTestDisableImport11
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkFindIntoSelectedPath
org.eclipse.che.selenium.miscellaneous.FindTextFeatureTest.checkFindByFileMask
org.eclipse.che.selenium.editor.CheckRestoringOfSplitEditor.checkRestoringStateSplittedEditor
org.eclipse.che.selenium.debugger.CppProjectDebuggingTest.shouldDebugCppProject
org.eclipse.che.selenium.plainjava.ConfigureSomeSourceFoldersTest.checkConfigureClasspathPlainJavaProject
org.eclipse.che.selenium.debugger.DebugExternalClassTest.shouldDebugJreClass
```

Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:535]: Also there is no section "Skipped tests" in the tests result page https://ci.codenvycorp.com/view/qa/job/che-integration-tests/52/. We doesn't have such statistics.

Comment (dmytro-ndp) [reply selenium/che-selenium-core/bin/webdriver.sh:535]: BTW: Jenkins places skipped tests in the list of failed tests. For example, skipped tests:
``` 
org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.shouldDebugStaticInnerClass
org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.shouldDebugLambdaExpressions
org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.shouldDebugAnonymousClass
```
screenshot from 2017-09-29 14 53 51

had been placed in the list: https://ci.codenvycorp.com/view/qa/job/che-integration-tests/53/
screenshot from 2017-09-29 14 56 46

**Review by tolusha — APPROVED** [review]

**Pull request merged**